CodePush.m

```diff
@@ -100,16 +109,13 @@ + (void)setUsingTestConfiguration:(BOOL)shouldUseTestConfiguration
 }
 
 /*
- * This is used to clean up all test updates. It can only be used
- * when the testConfigurationFlag is set to YES, otherwise it will
- * simply no-op.
+ * WARNING: This cleans up all downloaded and pending updates.
  */
-+ (void)clearTestUpdates
++ (void)clearUpdates
 {
-    if ([CodePush isUsingTestConfiguration]) {
-        [CodePushPackage clearTestUpdates];
-        [self removePendingUpdate];
-    }
+    [CodePushPackage clearUpdates];
```

Comment:
That's would prevent bundleURL from getting called after an update is installed, but the initial app load would still trigger bundeURL of the dev uses it for the JS bundle location. While debugging with Chrome and the packager, they wouldn't call bundleURL, so you're right, this would never get hit. Thanks for the clarification!